content/en/news/releases/1.13.x/announcing-1.13.3/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Announcing Istio 1.13.3
+linktitle: 1.13.3
+subtitle: Patch Release
+description: Istio 1.13.3 patch release.
+publishdate: 2022-04-18
+release: 1.13.3
+aliases:
+    - /news/announcing-1.13.3
+---
+
+This release contains bug fixes to improve robustness.
```

Comment:
As the release notes mentions two added supports, consider rewording "This release contains bug fixes to improve robustness" into "This release contains a few added supports and bug fixes to improve robustness."